js/geonovum/d3figures.js

```diff
@@ -0,0 +1,69 @@
+define(
+  ["geonovum/deps/d3",
+   "core/pubsubhub"],
```

Comment:
I included it for later, in case I needed to pub some error. I will just remove it.